### DISCUSSION TITLE
part unmerge

### DISCUSSION THREAD

**Discussion by sudingkun** [ClickHouse/ClickHouse#50649] [Q&A]
1、if some table merge part is big（eg：more than 30GiB ）and system.merges always merge other table。
so always see system.replication_queue  postpone_reason（Not executing log entry queue-0000481786 of type MERGE_PARTS for part 20230603_3432_4278_4 because source parts size (30GiB) is greater than the current maximum (xxx GiB).） 

2、if system.merges have a very big table （eg：system.trace_log） to merge，It takes up a lot of resources，Other tables cannot be merged

Is there anything I can do about it

**Comment by DerekChia** [comment]
1/ From previous issue (https://github.com/ClickHouse/ClickHouse/issues/48144), it looks like there is insufficient disk space for merge to happen. Do you see any other postpone_reason?

2/ How bad is the situation? How about trying to [stop merges](https://clickhouse.com/docs/en/sql-reference/statements/system#stop-merges) and then [start merges](https://clickhouse.com/docs/en/sql-reference/statements/system#start-merges) again? This comment might help - https://github.com/ClickHouse/ClickHouse/issues/12694#issuecomment-662886815

**Reply by sudingkun** [reply]
1、The same goes for other watches
2、I stop  system.trace_log merge and drop the partition，This will work。Is there a way to change some configuration so I don't have to do it manually every time
